x/wasm/types/validation.go

```diff
@@ -45,6 +46,12 @@ func ValidateLabel(label string) error {
 	if label != strings.TrimSpace(label) {
 		return ErrInvalid.Wrap("label must not start/end with whitespaces")
 	}
+	labelWithoutNonPrintableChars := strings.TrimFunc(label, func(r rune) bool {
+		return !unicode.IsGraphic(r)
+	})
+	if label != labelWithoutNonPrintableChars {
+		return ErrInvalid.Wrap("label must not have non printable characters")
```

Comment:
nit: let's avoid double negative ;-)
```suggestion
		return ErrInvalid.Wrap("label must have printable characters only")
```